lib/pacemaker/pcmk_sched_constraints.c

```diff
@@ -158,13 +158,13 @@ find_constraint_tag(const pcmk_scheduler_t *scheduler, const char *id,
     if (g_hash_table_lookup_extended(scheduler->tags, id,
                                      NULL, (gpointer *) tag)) {
         if (*tag == NULL) {
-            crm_warn("No resource is tagged with '%s'", id);
+            crm_notice("No resource is tagged with '%s'", id);
             return false;
         }
         return true;
     }
 
-    crm_warn("No template or tag named '%s'", id);
+    pcmk__config_warn("No resource, template, or tag named '%s'", id);
```

Comment:
pcmk__valid_resource_or_tag()

Reply:
Ah, the caller.

In general I don't like a log message referring to something that was done in the caller, in case the caller changes or new ones are added. In this case it's probably fine, since there's only one simple caller.

Up to you.